app/javascript/hyacinth_ui_v1/components/digital_objects/metadata/MetadataShow.js

```diff
@@ -25,11 +25,11 @@ class MetadataShow extends React.PureComponent {
       });
   }
 
-  renderField(dynamicField, data) {
+  renderField(dynamicField, data, elementKey) {
```

Comment:
Is this to remove warning around not having a unique key for each dynamic field?

Reply:
Right!

Reply:
I think I have forgotten how I implemented this, but is it possible to use something from the `dynamicField` hash?

Reply:
Good idea.  I'll use `dynamicField. stringKey` instead.